tutorials/R/Phenocam/phenocam-tools/getting-started-phenocamapi/getting-started-phenocamapi.Rmd

```diff
@@ -57,11 +57,10 @@ To explore the PhenoCam data, we'll use several packages for this tutorial.
 
```

Comment:
I couldn't comment on the line above,, but please add yourself as a contributor in the YAML header!